orgs/2gp.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Do we really need to introduce a new org shape? I think this will work fine on `prerelease` (right now, due to bugs), and in general, on `beta` or `release`.

Reply:
Yes, so we are consistent with the `2gp` org shape in Grants Management -- not for being a fix to the NPSP package version problem.